### PR TITLE
Dockerfile: upgraded to php 5.6.8

### PR DESCRIPTION
**Pull request by bryanlatten**
Also removed explicit, fragile versions of git and debug

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,11 +14,11 @@ RUN apt-get update && \
 RUN apt-get update && \
     apt-get install software-properties-common=0.92.37.3 -y
 
-# Install git with security updates
+# Install latest git with security updates
 # http://article.gmane.org/gmane.linux.kernel/1853266
 RUN add-apt-repository ppa:git-core/ppa -y && \
     apt-get update -yq && \
-    apt-get install -yq git=1:2.3.5-0ppa1~ubuntu14.04.1
+    apt-get install -yq git
```
Comment (chr0n1x) [inline Dockerfile:21]: does this install `>2.1`, or whatever the secure version is?

**Comment by ingluisjimenez** [comment]
:+1:

File: Dockerfile (hunk 1 above)
Comment (ingluisjimenez) [inline Dockerfile:21]: Should install latest

Comment (bryanlatten) [inline Dockerfile:21]: @chr0n1x latest from that upstream, older, explicit versions are `removed` quite frequently, and its a gigantic pain to maintain

File: Dockerfile [hunk 2]
```diff
@@ -30,18 +30,18 @@ RUN locale-gen en_US.UTF-8 && export LANG=en_US.UTF-8 && add-apt-repository ppa:
...
+        php5-intl=5.6.8+dfsg-1+deb.sury.org~trusty+1 \
+        php5-curl=5.6.8+dfsg-1+deb.sury.org~trusty+1 \
         php5-gearman=1.1.2-1+deb.sury.org~trusty+2 \
         php5-memcache=3.0.8-5+deb.sury.org~trusty+1 \
         php5-memcached=2.2.0-2+deb.sury.org~trusty+1 \
-        php5-mcrypt=5.6.7+dfsg-1+deb.sury.org~trusty+1 \
+        php5-mcrypt=5.6.8+dfsg-1+deb.sury.org~trusty+1 \
         php5-json=1.3.6-1+deb.sury.org~trusty+2 \
-        php5-xdebug=2.2.5-1+deb.sury.org~trusty+1 && \
+        php5-xdebug && \
```
Comment (jimdelois) [inline Dockerfile:44]: Do you happen to know which version of XDebug this installs?  Would be mint if it was `>= 2.3.2`.

**Pull request merged**